client/blocks/reader-full-post/index.jsx

```diff
@@ -353,14 +355,38 @@ export class FullPostView extends React.Component {
 	}
 }
 
+const ConnectedFullPostView = connect(
+	( state, ownProps ) => {
+		const {
+			site_ID: siteId,
+			feed_ID: feedId,
+			isExternal: is_external
```

Comment:
Is this the wrong way round? Looks like we should be mapping `is_external` on the `post` to the `isExternal` const.